### ISSUE TITLE
light-recessed-multiple

### ISSUE THREAD

**Issue by GitHubGoody** [Templarian/MaterialDesign#6841]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

Most of my house has recessed downlighting with multiple fixtures on each Lutron Caseta switch, so this icon is more accurate. I'll also use it for Home Assistant light groups.

### Share a preview of your contribution.

![image](https://user-images.githubusercontent.com/46235745/202210167-2acfafdc-4bcc-4f9e-b071-9df5f451def0.png)

I tried the preview generator, but when I joined all the paths, it messed up the design.

### Upload the icon source.

[light-recessed-multiple.zip](https://github.com/Templarian/MaterialDesign/files/10022753/light-recessed-multiple.zip)

**Comment by GitHubGoody** [comment]
@mririgoyen So...what does your thumbs down emoji mean? Do you have another idea?

**Comment by mririgoyen** [comment]
Sure, for a few reasons.

1. We already broke the guidelines by using perspective for this icon because there was no other way to easily discern the light type. 
2. The contribution doesn't follow our typical `-multiple` pattern. After spending a few minutes in Illustrator, is doesn't really work. So, I understand your approach.
3. The contribution breaks more guidelines by dropping the width of the lines below 2dp. Bumping them up to 2dp makes it impossible to fit more than one in the artboard.

Realistically, to add something like this would cause us to have to break even more guidelines on an icon we already reluctantly added for breaking one of the biggest guidelines: do not show perspective.

I understand the idea, the need, and the want for it, especially with other light icons we have with `-multiple` variants. However, I am strongly against adding this.

**Comment by GitHubGoody** [comment]
Well, maybe the guidelines need to be more flexible. As you said, clearly there is a use case for this (and probably other grouped lights).

Anyways, here is how HomeKit does it...as one alternative:

![image](https://user-images.githubusercontent.com/46235745/202851971-de7c8b0c-a2a2-46e4-b46d-259265c7b3c3.jpeg)

**Comment by mririgoyen** [comment]
🤣 We don't create the guidelines, Google does.

If you want to try and recreate the HoneKit layout with our base icon, feel free. Remember that the strokes need to be 2dp.

**Comment by GitHubGoody** [comment]
Meh, I'll bark up another tree. I thought this whole project was open sourced and just loosely based on Google guidelines. Close this if you want.

**Comment by mririgoyen** [comment]
It is open source, but we take the Material guidelines seriously. Otherwise, what's the point of being "Material Design Icons"? It actually took us years after the initial request for a recessed light to add it because we were so reluctant to break guidelines.

I do understand your frustration, though. Sometimes the concepts we would like don't translate well into the style. I also would have a potential use for this kind of icon in my Home Assistant instance, but the lack of it doesn't really bother me. If you're looking for this for Home Assistant, there's a pack on HACS called [custom-icons](https://github.com/Mariusthvdb/custom-icons) that would probably add your version in.

**Comment by GitHubGoody** [comment]
Yeah, I use ~3 other custom icon packages including that one. Since HA uses MDI as their default, I just wish THEY would automatically include ALL known icon sources in CORE so I didn't have to bother with maintaining them as custom "integrations." Ultimately, I couldn't care less where an icon I need comes from (or the guidelines it does/doesn't use), just that it's available, is recognizable, and fits my use case.

I will add, guidelines are meant to be bent and sometimes broken. ;-)

![image](https://user-images.githubusercontent.com/46235745/202861206-4715b1ce-063f-490d-a4d5-f08676ec4123.png)
